### PR TITLE
Add Plugin Manifest

### PR DESCRIPTION
**Pull request by JacobAGTyler**
This should enable the correct loading of the plugin, long term and to preempt the warning coming in 4.2. See discussion on CakePHP/CakePHP 4.1.2 bug fix in the long term. [CakePHP #14891](https://github.com/cakephp/cakephp/pull/14891)

### CODE REVIEW

**Comment by ADmad** [comment]
Changes for Cake 4 need to target the `cake-4.x` branch. `master` is for Cake 3.x and I don't intend to make new releases unless we hit an actual bug :)

**Comment by JacobAGTyler** [comment]
> Changes for Cake 4 need to target the `cake-4.x` branch. `master` is for Cake 3.x and I don't intend to make new releases for it unless we hit an actual bug :)

No problem, sorry wasn't looking at the branch. Just getting PHPBF running and I'll add that to the commit too.

**Comment by ADmad** [comment]
No worries, just rebase your bugfix branch on top of the `cake-4.x` branch.

**Comment by JacobAGTyler** [comment]
Sorry that was more of a faff than I intended. Should have just pulled it down from the beginning. Should both be all sorted now. Thanks for reviewing. :)

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Using [Composer][composer]:
 
 ```
 composer require muffin/footprint
+
```
Comment (ADmad) [inline README.md:19]: Unnecessary whitespace change.

Comment (JacobAGTyler) [reply README.md:19]: Sorry was more of an accident than by intent.
